src/replication.c

```diff
@@ -110,11 +111,11 @@ update_volstate(spec_t *spec)
  * and sets fd_closed also closes fd if read return 0, i.e., EOF
  * returns number of bytes read/written
  */
-static int64_t
+ssize_t
```

Comment:
should we change the caller functions variables also?

Reply:
All Caller function variables are signed integer or signed long int. I will change signed int to ssize_t.